Dev fp vs oop/README.md

```diff
@@ -0,0 +1,107 @@
+# Contrast functional patterns vs. OOP design patterns
+Sumiit have worked on the README file, but I want to do more with the code. Ashley suggested some areas to focus on. I want to make sure I'm helping the project in the right way so summit have worked on javascript in his part of contribution and i am working on JAVA 
+
+## Introduction
+In software development, we use different ways to solve problems. Two common methods are functional patterns and Object-Oriented Programming (OOP) design patterns. These methods help us organize our code and make it easier to understand and work with. Let's explore each of them to see how they work and what they do.
```

Comment:
You can remove the introduction section (since there is already an intro) if you take my overall advice and move your work to the [main README](https://github.com/nic-dgl104-winter-2024/pattern-library/blob/main/README.md).

Reply:
If you don't move it, I would recommend editing the Introduction section to make a clear distinction between function and OOP: You mention it above, but you sort of gloss over it, and you refer to both functional and OOP as "common [pattern] methods", which isn't quite accurate.